### ISSUE TITLE
Show code from app

### ISSUE THREAD

**Issue by tobrun** [mapbox/mapbox-android-demo#348]
One useful feature for the demo app is the ability to inspect the example code. This could be either directly in the app or with a redirect to the code on Github. I'm seeing two benefits of doing this:
 - makes code discoverable from within the app, users can easily take a peek at the used code.
 - makes UX around deep-linking in #347 better. When you deep link website content to an app, the content provided in the app should match what the website exposes (visual representation/working demo + code).

**Comment by langsmith** [comment]
I don't think this resolves this ticket, but just noting that the following pages exist.

- https://docs.mapbox.com/android/maps/examples/
- https://docs.mapbox.com/android/java/examples/
- https://docs.mapbox.com/android/plugins/examples/

 They are based on the examples in the demo app. Clicking on an example shows a video and the Github source code for that particular example.

**Comment by tobrun** [comment]
@langsmith  would be able to map example URLs to Activities in the app?

**Comment by langsmith** [comment]
yes, that's definitely a possibility. Will need help from @colleenmcginnis and/or other docs queens for that. 

The other thing I was thinking to resolve this ticket is to add a toolbar menu item, which means it'd be visible in every app example. Clicking on the button would kick the user to the phone's browser where the accompanying link on https://docs.mapbox.com/android/maps/examples/, https://docs.mapbox.com/android/java/examples/, or https://docs.mapbox.com/android/plugins/examples/ lives.

**Comment by colleenmcginnis** [comment]
>@langsmith would be able to map example URLs to Activities in the app?

Yep! I can set this up in /android-docs and add it to the update-examples script.

**Comment by langsmith** [comment]
>The other thing I was thinking to resolve this ticket is to add a toolbar menu item, which means it'd be visible in every app example. Clicking on the button would kick the user to the phone's browser where the accompanying link on https://docs.mapbox.com/android/maps/examples/, https://docs.mapbox.com/android/java/examples/, or https://docs.mapbox.com/android/plugins/examples/ lives.

This could lead to some funkiness because we usually take a little time to add a demo app example to the docs pages after the example gets added to the app. There would be no link to use in the toolbar menu item until the docs page is posted. We'd have to add the example to the app, eventually add the example to the docs pages, and then go _back_ into the app example to add the accompanying docs page URL as new PR that's separate from the original PR that adds the example to the app.

From the docs page to the Github demo app example is much better.
